code/modules/fishing/fishing_spots.dm

```diff
@@ -134,6 +161,18 @@ ABSTRACT_TYPE(/datum/fishing_spot)
 	/obj/item/reagent_containers/food/fish/real_goldfish = 5,\
 	/obj/item/reagent_containers/food/fish/red_herring = 1)
 
+/datum/fishing_spot/clown_shoes
+	fishing_atom_type = /obj/item/clothing/shoes/clown_shoes
+	rod_tier_required = 1
+	fish_available = list(/obj/item/reagent_containers/food/snacks/burger/moldy = 5, \
```

Comment:
Why do we have `/datum/fishing_lootpool/clown_shoes_loot` if we're adding more manually here? Sort of defeats the point imho, better to put these items in the lootpool and have as few manual assignments as possible.